src/Tizen.NUI/src/public/BaseComponents/ModelView.cs

```diff
@@ -15,202 +15,243 @@
+            internal static readonly int Texture0Url = Interop.ModelView.Texture0UrlGet();
+            internal static readonly int Texture1Url = Interop.ModelView.Texture1UrlGet();
+            internal static readonly int Texture2Url = Interop.ModelView.Texture2UrlGet();
+            internal static readonly int LightPosition = Interop.ModelView.LightPositionGet();
         }
 
-        public Model3dView() : this(Interop.Model3DView.Model3dViewNew(), true)
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ModelView() : this(Interop.ModelView.ModelViewNew(), true)
         {
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
 
         }
-        public Model3dView(string objUrl, string mtlUrl, string imagesUrl) : this(Interop.Model3DView.Model3dViewNew(objUrl, mtlUrl, imagesUrl), true)
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ModelView(string objUrl, string mtlUrl, string imagesUrl) : this(Interop.ModelView.ModelViewNew(objUrl, mtlUrl, imagesUrl), true)
         {
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
 
         }
-        public Model3dView(Model3dView model3dView) : this(Interop.Model3DView.NewModel3dView(Model3dView.getCPtr(model3dView)), true)
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ModelView(ModelView ModelView) : this(Interop.ModelView.NewModelView(ModelView.getCPtr(ModelView)), true)
         {
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
-        public Model3dView Assign(Model3dView model3dView)
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ModelView Assign(ModelView ModelView)
         {
-            Model3dView ret = new Model3dView(Interop.Model3DView.Model3dViewAssign(SwigCPtr, Model3dView.getCPtr(model3dView)), false);
+            ModelView ret = new ModelView(Interop.ModelView.ModelViewAssign(SwigCPtr, ModelView.getCPtr(ModelView)), false);
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
             return ret;
         }
 
-        public static Model3dView DownCast(BaseHandle handle)
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static ModelView DownCast(BaseHandle handle)
```

Comment:
I don't think `DownCast()` method is not required in C#. 
nui c# binding supports `as` and `is` keywords.
so I think it would be better to remove this method itself.